ddapm_test_agent/trace.py

```diff
@@ -273,7 +293,12 @@ def set_metric_tag(s: Span, k: str, v: MetricType) -> Span:
 
 def decode_v04(content_type: str, data: bytes) -> v04TracePayload:
```

Comment:
can we do these changes to the decode error logging separately. I'd like to have tests supporting it

Reply:
fine by me, I removed it